docker-compose-pool.yaml

```diff
@@ -31,11 +37,21 @@ services:
       - http_proxy
       - https_proxy
       - no_proxy
+    volumes:
+      - pool-1:/shared-pool-1
     expose:
       # KME listener port
       - 1948
     command: |
       bash -c "
+        while true;
+          do
+            if [ -e /shared-pool-1/wpe_mr_enclave.txt ];
+              then mv /shared-pool-1/wpe_mr_enclave.txt /project/avalon/wpe_mr_enclave.txt;
+              break;
+            fi;
+            sleep 1;
```

Comment:
Since it takes significant time to build cpp code in WPE, can we increase the sleep time say 3s?

Reply:
It's a while loop. So, should not matter enough.

Reply:
Ok